8-web-components/3-shadow-dom/article.md

```diff
@@ -44,7 +44,7 @@ Keyinchalik, [DOM spec](https://dom.spec.whatwg.org/#shadow-trees) va boshqa teg
 DOM elementi ikki xil DOM pastki daraxtiga ega bo'lishi mumkin:
 
 1. Light tree (yorug'lik daraxti) -- oddiy DOM pastki daraxti, HTML bolalaridan yaratilgan. Oldingi boblarda ko'rgan barcha pastki daraxtlar "light" edi.
```

Comment:
yengil ma'lumotlar tuzilmasi (tree)
tree - In computer science, a tree is a widely used abstract data type that represents a hierarchical tree structure with a set of connected nodes.